packages/components/src/components/checklist/checklist.jsx

```diff
@@ -21,25 +26,34 @@ const ItemStatus = ({ status, onClick }) => {
     }
 };
 
+ItemStatus.propTypes = {
+    button_text: PropTypes.string,
+    onClick: PropTypes.func,
+    status: PropTypes.string,
+};
+
 const Checklist = ({ items, className, itemClassName }) => (
     <div className={classNames('dc-checklist', className)}>
-        {items.map((item, idx) => (
-            <div
-                key={idx}
-                className={classNames('dc-checklist__item', itemClassName, {
-                    'dc-checklist__item--disabled': item.is_disabled,
-                })}
-            >
-                <div className='dc-checklist__item-text'>{item.content}</div>
-                <div
-                    className={classNames('dc-checklist__item-status', {
-                        'dc-checklist__item-status--disabled': item.is_disabled,
-                    })}
-                >
-                    <ItemStatus status={item.status} onClick={item.onClick} />
-                </div>
-            </div>
-        ))}
+        {items.map(
+            (item, idx) =>
+                item && (
```

Comment:
What is this check for?

Reply:
its to filter empty item

Reply:
Empty objects won't get caught by this though